server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -78,9 +82,20 @@ public Result index(Request request) {
             request, activeAndDraftPrograms, questionDefinitions, request.flash().get("success")));
   }
 
+  public Result seedCanonicalQuestions() {
```

Comment:
High-level: Can we put this logic into `seed` since we want these questions to be around even for dev instances?

If I understand correctly, that would mean we wouldn't need to call a separate URL from browser tests. I believe we call `/dev/seed` every time we start a Jest run.

Reply:
We do visit that URL at the beginning of the run, but only to call `clear`, the browser tests don't use the seed command.

I added the canonical questions to the existing seed action anyway since it doesn't hurt and we generally do want to ensure they're there.